.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Add `.DS_STORE` to your `.gitignore`. These are OSX metadata files used by Finder.